pkg/kubectl/scheme/install.go

```diff
@@ -106,7 +106,7 @@ func init() {
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:              admissionregistrationv1alpha1.GroupName,
-			RootScopedKinds:        sets.NewString("InitializerConfiguration", "ExternalAdmissionHookConfiguration"),
+			RootScopedKinds:        sets.NewString("InitializerConfiguration", "ValidatingWebhookConfiguration"),
```

Comment:
needs to list the mutating one, also?

Reply:
Ouch, is this the new scheme of kubectl? Do we really decided to rewrite the group installers here. This hurts.